frontend/src/scenes/project/Settings/index.js

```diff
@@ -78,7 +78,7 @@ function _Setup() {
                 </h2>
                 <p>
                     These are the domains and URLs where the Toolbar will automatically open if you're logged in. It's
-                    also where you'll be able to create Actions.
```

Comment:
Could we add a line about this behaviour lower down with the opt in itself as well. I'm sure this is going to be confusing otherwise

Reply:
Done, thanks!